### PR TITLE
Support skip API version check with new parameter --skipApiVersionCheck

### PR DESCRIPTION
**Pull request by blueww**
I have manually tested:

- Command: Azurite, azurite-blob, azurite-queue
- VSC: blob, queue

The test is : remove some API version from costants.ts, and send request to azurite with the removed API version:

- with --skipApiVersionCheck , will success
- without --skipApiVersionCheck, will fail

 

Thanks for contribution! Please go through following checklist before sending PR.

### PR Branch Destination

- For Azurite V3, please send PR to `dev` branch.
- For legacy Azurite V2, please send PR to `legacy-dev` branch.

### Always Add Test Cases

Make sure test cases are added to cover the code change.

### Development Guideline

Please go to CONTRIBUTION.md for steps about setting up development environment and recommended Visual Studio Code extensions.

### CODE REVIEW

**Comment by XiaoningLiu** [comment]
@blueww HTTPs test case failed, can you check oauth parameters are corectly set in test cases? As this PR moves OAuth parameter order.

**Comment by XiaoningLiu** [comment]
> export const QUEUE_API_VERSION = "2019-07-07";

Bump up API version here.

Same with blob

---
Refers to: src/queue/utils/constants.ts:4 in 262a2a3. [](commit_id = 262a2a30e7a75c807ec86cf5d7f50eacc64647fc, deletion_comment = False)

**Comment by XiaoningLiu** [comment]
> export const VERSION = "3.7.0";

Bump up SDK version.
Same with blob

---
Refers to: src/queue/utils/constants.ts:3 in 262a2a3. [](commit_id = 262a2a30e7a75c807ec86cf5d7f50eacc64647fc, deletion_comment = False)

**Comment by XiaoningLiu** [comment]
>   "version": "3.7.0",

Bump up SDK version here.

---
Refers to: package.json:6 in 262a2a3. [](commit_id = 262a2a30e7a75c807ec86cf5d7f50eacc64647fc, deletion_comment = False)

File: README.md [hunk 1]
```diff
@@ -177,6 +178,8 @@ Above command will try to start Azurite image with configurations:
 
 `--loose` enables loose mode which ignore unsupported headers and parameters.
 
+`--skipApiVersionCheck` skip the request API version check.
```
Comment (XiaoningLiu) [inline README.md:181]: >skipApiVersionCheck [](start = 3, length = 19)

Update mcr readme.md.  Find this file under workspace root.

Comment (blueww) [reply README.md:181]: updated. and fixed all other comments for this PR.

**Pull request merged**